aten/src/ATen/native/TensorTransformations.h

```diff
@@ -10,8 +10,11 @@ namespace at {
 namespace native {
 
 static inline void flip_check_errors(int64_t total_dims, int64_t flip_dims_size, IntArrayRef dims) {
+  if (flip_dims_size==0) {
```

Comment:
`if (flip_dims_size == 0) {` - spaces around the `==` operator

Reply:
should I add this? the pull request is already approved new commit will not have any problem right?

Reply:
It's OK, you don't need to change it.